src/components/Auth/Auth.tsx

```diff
@@ -38,6 +47,27 @@ function Auth() {
     setSignup(!signup);
   }, [signup]);
 
+  const handleRequestPasswordReset = useCallback(async () => {
+    setResetting(true);
+    await UserAPI.requestResetPassword({ login: pwResetLogin });
+    showPwResetNotification();
```

Comment:
you need to set `resetting` back to `false` at the end. I even linked you to an example so it should be easy enough to follow. the flow for forms is always the same:
- when submit clicked, set `loading` or equivalent variable to `true` and `try` the stuff below
- validate form
- do whatever else
- in `finally` set `loading` back to `false`
- use `loading` to disable form inputs and the submit button